azure-pipelines.yml

```diff
@@ -192,6 +192,8 @@ jobs:
   - bash: ci/scripts/verible-lint.sh fpv
     condition: eq(variables['Build.Reason'], 'PullRequest')
     displayName: Verible FPV (Verilog lint)
+  - bash: ci/bazelisk.sh run //util/py/scripts:audit_sec_mmio_calls -- --log-level info
+    displayName: Audit calls to sec_mmio functions
```

Comment:
This is probably not worth running in CI, but I wanted to demonstrate that it works at least temporarily.

@milesdai I have to correct my earlier claim that `--nobuild` would speed up the audit. It turns out it does not selectively skip build actions while allowing aspect actions to run. It seems to skip both!

Reply:
Yeah, I agree, this should probably a part of our final checklist though. See https://github.com/lowRISC/opentitan/blob/master/sw/device/silicon_creator/rom/data/rom_manual_testplan.hjson and https://github.com/lowRISC/opentitan/blob/master/util/run_manual_tests.py.

Reply:
Sounds good. Filed https://github.com/lowRISC/opentitan/issues/19288 to keep track of this.